lib/common.sh

```diff
@@ -139,8 +139,12 @@ export FORCE_REPO_UPDATE="${FORCE_REPO_UPDATE:-true}"
 
 export M3PATH="${M3PATH:-${GOPATH}/src/github.com/metal3-io}"
 export BMOPATH="${BMOPATH:-${M3PATH}/baremetal-operator}"
-export BMOREPO="${BMOREPO:-https://github.com/metal3-io/baremetal-operator.git}"
-export BMO_BASE_URL="${BMO_BASE_URL:-metal3-io/baremetal-operator}"
+# export BMOREPO="${BMOREPO:-https://github.com/metal3-io/baremetal-operator.git}"
```

Comment:
Remove the commented out lines if they're not necessary.

Reply:
I have kept the old lines because I thought that I need to revert to them after the testing is done. I am now testing with a PR that is not yet merged as you might notice.

Reply:
If you need to test with BMO from your branch, put the required config to `config_example.sh` as that is always sourced, then you don't need to mix test config to actual changes.

Reply:
Gotcha! Thanks for the explanation.